flow-typed/forms.js

```diff
@@ -354,6 +356,14 @@ declare type CivicaStreetNameElement = {
   placeholderValue?: string,
 } & LookupFormElement
 
+declare type CivicaNameRecordElement = {
+  type: 'civicaNameRecord',
+  readOnly: boolean,
+  defaultValue?: CivicaNameRecord,
+  placeholderValue?: string,
```

Comment:
which element will this apply to? Should we add placeholders for all the elements or remove this?